packages/manager/apps/pci-object-storage/src/pages/object-storage/storage/s3Id/S3.layout.tsx

```diff
@@ -40,8 +40,17 @@ export function breadcrumb() {
   return <S3Name />;
 }
 
+const HIDE_HEADER_ROUTE_PATTERNS = ['s3.object.'];
 const HIDE_TABS_ROUTE_PATTERNS = ['s3.object.'];
 
+const shouldHideS3Header = (
+  matches: ReturnType<typeof useMatches>,
+): boolean => {
+  return matches.some((match) =>
+    HIDE_HEADER_ROUTE_PATTERNS.some((pattern) => match.id?.startsWith(pattern)),
+  );
+};
+
```

Comment:
This will be the exact same result as shouldHides3Tabs, as the pattern is the same. Any reason why we want to duplicate here ?